client/src/components/Projects/ProjectTableRow.js

```diff
@@ -82,12 +82,7 @@ const ProjectTableRow = ({
   // deleted, in which case it will show the deleted date followed by "-Deleted" in red.
   const dateModifiedDisplay = () => {
     if (project.dateTrashed) {
-      return (
-        <span>
-          {formatDate(project.dateTrashed)}
-          <span style={{ color: "red" }}>-Deleted</span>
-        </span>
-      );
+      return <span>{formatDate(project.dateTrashed)}</span>;
```

Comment:
I don't know why you moved the deleted indicator to the visibility column. Maybe you misunderstood our conversation last week where I pointed out that the Date Modified column is where you would see that the deleted date is displayed if the project is deleted, per Emily's request, but we don't want to move it to the Status column.

Reply:
For this one, I refered to the figma and checklist of action items.

#1646 

![Screenshot 2024-09-04 at 10 39 47 AM](https://github.com/user-attachments/assets/cfe4d0a6-549a-4533-bfa2-9979cb59e914)

Link to prototype:

https://www.figma.com/proto/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?type=design&node-id=12173-2052&t=wQuxLI1JbOw0OzLn-1&scaling=scale-down&page-id=2499%3A11472&starting-point-node-id=11997%3A885